_content/tour/rus/algorithms-data.article

```diff
@@ -85,15 +85,15 @@ This sample program implements a basic circular queue.
 
 .play algorithms/data/queue_circular.go
 
-* Stack
+* Стек
 
-This sample program implements a basic stack.
+Данный пример программы реализует базовый стек.
 
-- See more at [[https://en.wikipedia.org/wiki/Stack_(abstract_data_type)][https://en.wikipedia.org/wiki/Stack_(abstract_data_type)]]
+- См. подробнее на [[https://en.wikipedia.org/wiki/Stack_(abstract_data_type)]][https://en.wikipedia.org/wiki/Stack_(abstract_data_type)]].
 
-*Diagram*
+*Диаграмма*
 
-    With a stack, the first value in is the last value out.
+    При использовании стека первое входящее значение становится последним исходящим.
```

Comment:
```suggestion
    В стеке первое входящее значение является последним выходящим значением.
```

So it uses the same style as queue's definition (just like in the English version)